src/core/engines/BaseEngine.sol

```diff
@@ -176,12 +183,36 @@ abstract contract BaseEngine is ReentrancyGuard {
         optionToken.mint(recipient, tokenId, amount);
     }
 
+    /**
+     * @dev mint option token into account, increase short position (debt) and increase long position in account memory
+     * @param _data bytes data to decode
```

Comment:
The comment seems false: it's creating long position in another account in storage directly, not in memory

Reply:
Agreed on the "long position" part. 

The "memory" vs "storage" seems to be the case for many comments in this section. Unless i am not understanding your use of the words. 

For example `_mintOption` the comment is `* @dev mint option token to user, increase short position (debt) in account memory` the increased short position is made in account storage, not in memory.

Reply:
You're right, that's a wrong comment. Will need to clean it up